prisma/schema.prisma

```diff
@@ -19,6 +19,7 @@ model Vote {
   id          String @id @unique @default(uuid())
   fingerprint String
   postId      String
+  value       Int    @default(0)
```

Comment:
gibt es einen Grund, warum du nicht die in-build enums benutzt?
https://www.prisma.io/docs/concepts/components/prisma-schema/data-model